demos/OH-EXF-Kamakura/defaults.mk

```diff
@@ -15,17 +15,17 @@ export DOCKER_EDGEX_ORG ?= edgexfoundry
 export EDGEX_NEXUS_REGISTRY ?= nexus3.edgexfoundry.org:10004
 export DOCKER_EDGEX_NEXUS_ORG ?=
 
-# git url to clone for the device-onvif-camera source code
-export REPO_URL_device_onvif_camera ?= https://github.com/edgexfoundry-holding/device-onvif-camera.git
-# branch or tag name for device-onvif-camera
-export TAG_device_onvif_camera ?= main
-
 # git url to clone for the model downloader tool
 export REPO_URL_model_downloader ?= https://github.com/intel/edge-video-analytics-microservice.git
 # branch or tag name for model downloader tool
 export TAG_model_downloader ?= v0.7.0
 # full image name and version of openvino data image to use when downloading and converting models
 export OPENVINO_DATA_IMAGE ?= openvino/ubuntu20_data_dev:2021.4.2
 
+# git url to clone for the edgex-examples source code
+export REPO_URL_edgex_examples ?= https://github.com/EdgeX-Camera-Management/edgex-examples.git
```

Comment:
Shouldn't this be from actual edgexfoundry repo `https://github.com/edgexfoundry/edgex-examples/tree/main/application-services/custom/camera-management` instead of this?

Reply:
It can be now, but at the time of creation, the patches were not merged into the main repo.